src/layouting/data_limits.jl

```diff
@@ -121,7 +121,7 @@ function atomic_limits(x::Text{<: Tuple{Arg1}}) where Arg1
             end
         else
             if x.position[] isa Union{StaticArrays.StaticArray, Tuple{Real, Real}, GeometryBasics.Point}
-                bb = FRect3D(x.position[])
+                bb = frect3d(x.position[])
```

Comment:
I think we should try to not to introduce these kind of secondary, abstractplotting specific APIs. I know I'm guilty of this a lot... And I learned, that it's very annoying to clean up later and explain to other contributors. 
If there's something wrong with FRect3D we should fix it in GeometryBasics - or introduce a proper boundingbox function ;) But not just introduce a new function, with almost the same name, which (probably?) behaves slightly different.

Reply:
I agree. I think it's not very clear what the API is for these objects and I can often not find what I need quickly enough. So I write these helper functions instead of stopping to make a PR somewhere else.

What I also find super annoying is that multiplication of rectangles, addition with points, and practical things like that are either not defined or have such strong type restrictions that I can't use them without doing a lot of guard rail code.

Plus, that I can't do `array_of_points .+ point` because a point behaves as a small array. That's super impractical..

Reply:
Yeah, we should fix that: https://github.com/JuliaGeometry/GeometryBasics.jl/blob/master/src/rectangles.jl#L271

The types look pretty unrestricted? 

>Plus, that I can't do array_of_points .+ point because a point behaves as a small array. That's super impractical..

Yes, totally agree... I think I'm ready to have that fixed as well^^